packages/wp-source/src/index.ts

```diff
@@ -15,9 +15,10 @@ const wpSource: WpSource = {
     source: {
       api: new Api(),
       resolver: new Resolver(),
-      populate
+      populate,
+      routeUtils
```

Comment:
For easy access I'd spread `...routeUtils` to get:
`libraries.source.routeToParams` vs `libraries.source.routeUtils.routeToParams`.